drivers/SmartThings/matter-appliance/src/init.lua

```diff
@@ -131,8 +177,22 @@ local function do_configure(driver, device)
     end
     device.log.info_with({hub_logs=true}, string.format("Updating device profile to %s.", profile_name))
     device:try_update_metadata({profile = profile_name})
+  elseif extractorHood.can_handle({}, driver, device) then
+    local profile_name = "extractor-hood"
+    if #hepa_filter_eps > 0 and #ac_filter_eps > 0 then
+      profile_name = profile_name .. "-hepa" .. "-ac"
+    elseif #hepa_filter_eps > 0 then
+      profile_name = profile_name .. "-hepa"
+    elseif #ac_filter_eps > 0 then
+      profile_name = profile_name .. "-ac"
+    end
```

Comment:
nit: can't this be just two if statements? Like a #hepa_filter_eps, and then a #ac_filter_eps check?

Reply:
Updated as suggested. Also made this change for the logic above this